src/ServicePulse.Host/app/lib/bootstrap-3.3.5/css/bootstrap.css

```diff
@@ -1583,8 +1583,8 @@ pre code {
   overflow-y: scroll;
 }
 .container {
-  padding-right: 0;
-  padding-left: 0;
+  padding-right: 15px;
+  padding-left: 15px;
```

Comment:
Ah! I remember the discussion, but missed that last comment.

I checked the changes and it's all good, so 👍